genome_sampler/combine.py

```diff
@@ -2,7 +2,7 @@
 
 import qiime2
 
-from genome_sampler.common import IDSelection
+from q2_types._format import IDSelection
```

Comment:
We should avoid importing from `_format` as the `_` files are private by convention (so no guarantee that this import will continue to work in the future).